zipkin-storage/influxdb-http/src/main/java/zipkin/storage/influxdb/http/InfluxDBHttpStorage.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright 2016-2017 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package zipkin.storage.influxdb.http;
+
+import org.influxdb.InfluxDB;
+import zipkin2.Call;
+import zipkin2.DependencyLink;
+import zipkin2.Span;
+import zipkin2.storage.QueryRequest;
+import zipkin2.storage.SpanConsumer;
+import zipkin2.storage.SpanStore;
+import zipkin2.storage.StorageComponent;
+
+import java.util.List;
+
+public class InfluxDBHttpStorage extends StorageComponent implements SpanStore, SpanConsumer {
```

Comment:
long term, you'll probably need to decouple these (ex not implement SpanStore or SpanConsumer rather provide instances of them)